src/Components/BookList.js

```diff
@@ -1,29 +1,33 @@
 import React from 'react';
-import { PropTypes } from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { removebook } from '../redux/books/booksSlice';
+
+function BookList() {
+  const books = useSelector((store) => store.books.book);
+  const dispatch = useDispatch();
 
-function BookList({ booksL }) {
   return (
     <ul>
 
       {
-        booksL.map((books) => (
-          <li key={books.id}>
+        books.map((book) => (
+          <li key={book.id}>
             <div className="bookInfo">
 
               <div className="bookDetails">
                 <div className="category">
-                  {books.category}
+                  {book.category}
                 </div>
                 <div className="title">
```

Comment:
**OPTIONAL** - There's a huge X-scroll on your application and this does not make for a good UI/UX. Please fix this 🙏